README.md

```diff
@@ -3,70 +3,33 @@ status: draft
-
-**Preserve Project Identity:** Even as trademarks and domain names integrate with CF, projects retain their distinct identity, celebrated and recognized globally.
-
-**Transparency and Longevity:** We value clear governance and are committed to the sustainability of projects, evident in aspects like our [Succession Planning][succession].
-
-**Empower, Don't Enforce:** CF's mission centers on empowerment without ensnaring projects in restrictive barriers.
-
-## CF's Focus and Vision
-
-**Prioritize Action:** We favor tangible outcomes over extended discussions, concentrating on:
-
-- **Application Frameworks:** For efficient app development and deployment.
-- **Libraries:** Essential tools that minimize application complexities.
-
-**Selective Alignment:** Not every project aligns with CF. While we respect the importance of *Specifications & Standards*, we do not engage in their definition or debate. This is a home for **self-governing, code-centric projects**.
+CF's action-oriented approach prioritizes **Application Frameworks** and **Libraries** that are critical for developers. We support self-governing, code-centric projects, steering clear of specifications and standards debates to focus on tangible outcomes.
 
 ## Community and Sponsorship
 
-CF is committed to inclusive growth, opening its doors to all individuals who share our vision, not just code contributors. We encourage participation from those interested in supporting and steering our projects, offering them a voice and a vote in our governance. For detailed information on membership and eligibility, refer to our [Bylaws][].
-
-**Collaboration and Voice**: Our community thrives on cooperation and diverse perspectives. Here, every member, including project leaders and supporters, has the opportunity to contribute to decision-making processes.
+CF's doors are open to everyone passionate about open-source innovation. We invite all individuals, not just code contributors, to participate and help steer our projects. Our [Bylaws][] detail membership and governance, emphasizing collaboration, diverse perspectives, and advisory input from our [Advisory Board][cfab] to align sponsorship with community interests.
 
-**Advisory Input**: While sponsors do not have voting rights, they play a vital role through the [Advisory Board][cfab], offering strategic insights that guide the foundation's direction. This structure ensures that sponsorship is aligned with community interests and open-source integrity.
+## Get Involved
 
-## Additional resources
+Advance open-source innovation and collaboration with us.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Commonhaus Foundation Governance](bylaws/README.md)
-- [Succession Planning][succession]
-- [Security Policy][sec-policy]
-- [IP Policy][ip-policy]
+Explore how to contribute through our [Code of Conduct](CODE_OF_CONDUCT.md), [Governance Bylaws][Bylaws], and [Policies][].
```

Comment:
Should this also refer to the contribution guide in the root?

Reply:
oh. now I see this. 🤦‍♀️ yes..